tests/stress/iampartialpolicy_stress_test.go

```diff
@@ -0,0 +1,177 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/GoogleCloudPlatform/k8s-config-connector/config/tests/samples/create"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// TestStressIAMPartialPolicy runs a stress test that checks behaviour with a lot of IAMPartialPolicy objects.
+func TestStressIAMPartialPolicy(t *testing.T) {
+	if os.Getenv("RUN_STRESS") == "" {
```

Comment:
I may miss something, but I don't see the RUN_STRESS is setup or called in KCC presubmit. Do you mind elaborating or commenting a little bit more.

Reply:
oh, I see. It will be a new config like `E2E_KUBE_TARGET=envtest RUN_E2E=1 E2E_GCP_TARGET=mock` and not yet triggered in github CI